build.gradle

```diff
@@ -290,6 +290,16 @@ subprojects {
     }
   }
 
+  // Exclude tests which are known to be flaky in the Travis CI environment
+  if (System.getenv('TRAVIS') == 'true' && System.getenv('USER') == 'travis') {
```

Comment:
Should we check System.getenv('TRAVIS') != null first?

Reply:
Doesn't this check accomplish that? `null != 'true'`, and there's no risk of NPE